modules/security-container-content-external-scanning.adoc

```diff
@@ -0,0 +1,262 @@
+  "timestamp": "2016-09-08T05:04:46Z",
+  "reference": "https://www.open-scap.org/930492",
+  "compliant": true,
+  "scannerVersion": "1.2",
+  "summary": [
+    { "label": "critical", "data": "4", "severityIndex": 3, "reference": null },
+    { "label": "important", "data": "12", "severityIndex": 2, "reference": null },
+    { "label": "moderate", "data": "8", "severityIndex": 1, "reference": null },
+    { "label": "low", "data": "26", "severityIndex": 0, "reference": null }
+  ]
+}
+----
+
+This example shows the
+link:https://catalog.redhat.com/software/containers/explore[Red Hat Container Catalog]
+annotation for an image with health index data
+with an external URL for additional details:
+
+.Red Hat Container Catalog annotation
+[source,json]
+----
+{
+  "name": "Red Hat Container Catalog",
+  "description": "Container health index",
+  "timestamp": "2016-09-08T05:04:46Z",
+  "reference": "https://access.redhat.com/errata/RHBA-2016:1566",
+  "compliant": null,
+  "scannerVersion": "1.2",
+  "summary": [
+    { "label": "Health index", "data": "B", "severityIndex": 1, "reference": null }
+  ]
+}
+----
+
+[id="security-annotating-image-objects_{context}"]
+== Annotating image objects
+
+While image stream objects
+are what an end user of {product-title} operates against,
+image objects are annotated with
+security metadata. Image objects are cluster-scoped, pointing to a single image
+that may be referenced by many image streams and tags.
+
+[id="security-example-annotate-CLI_{context}"]
+=== Example annotate CLI command
+
+Replace `<image>` with an image digest, for example
+`sha256:401e359e0f45bfdcf004e258b72e253fd07fba8cc5c6f2ed4f4608fb119ecc2`:
+
+----
+$ oc annotate image <image> \
+    quality.images.openshift.io/vulnerability.redhatcatalog='{ \
+    "name": "Red Hat Container Catalog", \
+    "description": "Container health index", \
+    "timestamp": "2020-06-01T05:04:46Z", \
+    "compliant": null, \
+    "scannerVersion": "1.2", \
+    "reference": "https://access.redhat.com/errata/RHBA-2020:2347", \
+    "summary": "[ \
+      { "label": "Health index", "data": "B", "severityIndex": 1, "reference": null } ]" }'
+----
+
+[id="controlling-pod-execution_{context}"]
+== Controlling pod execution
+
+Use the `images.openshift.io/deny-execution` image policy 
+to programmatically control if an image can be run.
+
+[id="security-controlling-pod-execution-example-annotation_{context}"]
+=== Example annotation
+
+[source,yaml]
+----
+annotations:
+  images.openshift.io/deny-execution: true
+----
+
+[id="security-integration-reference_{context}"]
+== Integration reference
+
+In most cases, external tools such as vulnerability scanners develop a
+script or plug-in that watches for image updates, performs scanning, and
+annotates the associated image object with the results. Typically this
+automation calls the {product-title} {product-version} REST APIs to write the annotation. See
+the REST API Reference for general
```

Comment:
This still uses the old "REST API Reference" title

Reply:
The correct name now seems to be "{product-title} REST APIs" so that's what I changed it to.